uber/site_sections/preregistration.py

```diff
@@ -98,6 +98,48 @@ def index(self, message=''):
     def dealer_registration(self, message=''):
         return self.form(badge_type=c.PSEUDO_DEALER_BADGE, message=message)
 
+    def redeem(self, session, message='', **params):
+        accepted_message = 'Promo Code Applied!'
+        rejected_message = 'Promo Code Rejected!'
+        if 'promo' and 'edit_id' in params:
+            destroy_promo_code = False
+
+            attendee = cherrypy.session['unpaid_preregs'][params['edit_id']]
+            if attendee is not None:
+                pc = session.query(PromoCode).filter(PromoCode.code == params['promo']).first()
```

Comment:
Yeah, I've been guilty of this myself a lot.  "What do you mean ``acv`` isn't an intuitive variable name?  Just look at the surrounding context and it's obvious!"